_includes/_livestream.html

```diff
@@ -3,58 +3,97 @@
+    let isStream = true; // TODO: set this back to false.
+    let playerUrl = '';
+    let entryId = '';
+    let currentPageTitle = "{{ page.name }}";
 
-  livestreamData.forEach(function (livestream) {
-    let startTime = new Date(livestream.startTime).getTime();
-    let endTime = new Date(livestream.endTime).getTime();
+    livestreamData.forEach(function (livestream) {
+      let startTime = new Date(livestream.startTime).getTime();
+      let endTime = new Date(livestream.endTime).getTime();
 
-    let isPageMatch = livestream.pages.some(function (page) {
-      return page === currentPageTitle || page === 'Churchwide (all location pages)';
-    });
+      let isPageMatch = livestream.pages.some(function (page) {
+        return page === currentPageTitle || page === 'Churchwide (all location pages)';
+      });
 
-    if (isPageMatch && isTimeInRange(startTime, endTime)) {
-      playerUrl = livestream.url || 'https://resi.media/cr-cinc-oh/Manifest.m3u8';
-      entryId = livestream.id || 'VideoManager';
-      isStream = true;
+      if (isPageMatch && isTimeInRange(startTime, endTime)) {
+        playerUrl = livestream.url || 'https://resi.media/cr-cinc-oh/Manifest.m3u8';
+        entryId = livestream.id || 'VideoManager';
+        isStream = true;
 
-      // Refresh the page one minute after the livestream starts
-      let refreshStartTime = startTime + 60000;
-      if (refreshStartTime > Date.now()) {
-        setTimeout(function () {
-          location.reload(true);
-        }, refreshStartTime - Date.now());
-      }
+        // Refresh the page one minute after the livestream starts
+        let refreshStartTime = startTime + 60000;
+        if (refreshStartTime > Date.now()) {
+          setTimeout(function () {
+            location.reload(true);
+          }, refreshStartTime - Date.now());
+        }
 
-      // Refresh the page one minute after the livestream ends
-      let refreshEndTime = endTime + 60000;
-      if (refreshEndTime > Date.now()) {
-        setTimeout(function () {
-          location.reload(true);
-        }, refreshEndTime - Date.now());
+        // Refresh the page one minute after the livestream ends
+        let refreshEndTime = endTime + 60000;
+        if (refreshEndTime > Date.now()) {
+          setTimeout(function () {
+            location.reload(true);
+          }, refreshEndTime - Date.now());
+        }
       }
-    }
-  });
+    });
 
-  if (isStream) {
-    let resiPlayerSection = document.createElement('section');
-    resiPlayerSection.className = 'bg-blue';
-    resiPlayerSection.id = 'resi-player';
+    if (isStream) {
+      let livestreamBanner = document.createElement('div');
+      livestreamBanner.className = 'width-100';
+      livestreamBanner.style.cssText = 'width: 100vw; margin: -80px 0 0; padding: 8px 0; text-align: center; background-color: black;';
+      livestreamBanner.innerHTML = `
+        <div style="position: relative; z-index: 10; margin-bottom: 120px;">
```

Comment:
This is also overlapping the shared header due to the z-index

<img width="763" alt="Screen Shot 2024-11-13 at 8 41 36 AM" src="https://github.com/user-attachments/assets/4d1b9ceb-03f8-4852-8429-e4a8685fa2af">